### DISCUSSION TITLE
Map container port to all interfaces on host

### DISCUSSION THREAD

**Discussion by shockogit** [containers/podman#22065] [General]
Running podman 4.9.3 on Windows 10 Enterprise 22 H2 with latest WSL 2

```
WSL version: 2.1.5.0
Kernel version: 5.15.146.1-2
WSLg version: 1.0.60
MSRDC version: 1.2.5105
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.19045.4046
```

I'm trying to map a port on a container using -p for example -p 8080:8080 but when the container runs the port on the host is bound to 127.0.0.1. I need it to bind to 0.0.0.0. How can this be achieved?

**Comment by Luap99** [comment]
This is a limitation of the WSL 2  network stack,see https://learn.microsoft.com/en-us/windows/wsl/networking#accessing-a-wsl-2-distribution-from-your-local-area-network-lan

**Reply by shockogit** [reply]
Thanks @Luap99 . So the docker runtime handled this for us it seems (i.e. it must have created the OS level port forwarding). I'll add manually.

**Reply by Luap99** [reply]
Feel free to create a feature request for this, if docker can handle it we should likely support that too.
